consensus/pacemaker/pacemaker.go

```diff
@@ -9,58 +9,58 @@ import (
 	consensusTelemetry "github.com/pokt-network/pocket/consensus/telemetry"
 	typesCons "github.com/pokt-network/pocket/consensus/types"
 	"github.com/pokt-network/pocket/runtime/configs"
+	"github.com/pokt-network/pocket/shared/codec"
 	"github.com/pokt-network/pocket/shared/modules"
+	"google.golang.org/protobuf/types/known/anypb"
 )
 
 const (
+	DefaultLogPrefix    = "NODE"
 	pacemakerModuleName = "pacemaker"
-	timeoutBuffer       = 30 * time.Millisecond // A buffer around the pacemaker timeout to avoid race condition; 30ms was arbitrarily chosen
+
+	// A buffer around the pacemaker timeout to avoid race condition; 30ms was arbitrarily chosen
+	timeoutBuffer = 30 * time.Millisecond
+
+	NewRound = typesCons.HotstuffStep_HOTSTUFF_STEP_NEWROUND
```

Comment:
Do these constants (and the one above) need to be capitalized (i.e. exposed outside of this module) or can we scope them down?